.github/workflows/ci.yml

```diff
@@ -62,6 +62,19 @@ jobs:
         run: yarn install --non-interactive --check-files --frozen-lockfile
       - name: Build library
         run: ./node_modules/.bin/gulp build
+      - name: Install github changelog generator
+        run: sudo gem install github_changelog_generator
+      - name: Export github token
+        run: export CHANGELOG_GITHUB_TOKEN=${{secrets.GITHUB_TOKEN}}
+      - name: Build release notes
+        run:  ./node_modules/.bin/gulp generateChangelog
+      - name: Write github token to file
+        run: |
+          echo "${{ secrets.GITHUB_TOKEN }}" > gitToken.txt
+      - name: Authenticate for changelog
+        run: gh auth login --with-token < gitToken.txt
```

Comment:
I don't think you have to write the token to a file in order to read it in again. Shouldn't this work:

~~~
echo "${{ secrets.GITHUB_TOKEN }}" | gh auth login --with-token
~~~
?

Reply:
Hmm, not sure, let me try

Reply:
That worked, thanks!